games.yaml

```diff
@@ -174,6 +174,16 @@
       media:
         - image: http://xarchon.seul.org/img/snap-1.gif
         - image: http://xarchon.seul.org/img/snap-2.gif
+        
+- name: "Area 51 (2005, Midway)"
```

Comment:
You have to write a name here which resolves to a page on Wikipedia. In your case I think it should be `[Area 51, Area 51 (2005 video game)]`.